python/node_classification.py

```diff
@@ -0,0 +1,641 @@
+
+
+# other necessary parameters
+class OtherParams:
+    DEVICE_TYPE = "device_type"
+    PATH_TO_MODEL = "path_to_model"
+    PATH_TO_MODEL_LAST = "path_to_model_last"
+    PATH_TO_MODEL_SECOND_LAST = "path_to_model_second_last"
+    PATH_TO_MODEL_THIRD_LAST = "path_to_model_third_last"
+
+
+# main variables for modelling
+class Modelling:
+    # all None until set_params are executed
+    data: mgp.Any = None
+    # model: InductiveModel = None
+    model: mgp.Any = None
+    opt = None
+    criterion = None
+
+
+# list for saving logged data
+logged_data: mgp.List = []
+
+# dictionary of defined input types
+DEFINED_INPUT_TYPES = {
+    ModelParams.HIDDEN_FEATURES_SIZE: list,
+    ModelParams.LAYER_TYPE: str,
+    TrainParams.NUM_EPOCHS: int,
+    OptimizerParams.LEARNING_RATE: float,
+    OptimizerParams.WEIGHT_DECAY: float,
+    DataParams.SPLIT_RATIO: float,
+    MemgraphParams.NODE_ID_PROPERTY: str,
+    OtherParams.DEVICE_TYPE: str,
+    TrainParams.CONSOLE_LOG_FREQ: int,
+    TrainParams.CHECKPOINT_FREQ: int,
+    TrainParams.BATCH_SIZE: int,
+    ModelParams.AGGREGATOR: str,
+    DataParams.METRICS: list,
+    HeteroParams.OBSERVED_ATTRIBUTE: str,
+    HeteroParams.FEATURES_NAME: str,
+    HeteroParams.CLASS_NAME: str,
+    HeteroParams.REINDEXING: dict,
+    HeteroParams.INV_REINDEXING: dict,
+    OtherParams.PATH_TO_MODEL: str,
+    OtherParams.PATH_TO_MODEL_LAST: str,
+    OtherParams.PATH_TO_MODEL_SECOND_LAST: str,
+    OtherParams.PATH_TO_MODEL_THIRD_LAST: str,
+}
+
+# dictionary of default values for input types
+DEFAULT_VALUES = {
+    ModelParams.HIDDEN_FEATURES_SIZE: [16],
+    ModelParams.LAYER_TYPE: "SAGE",
+    TrainParams.NUM_EPOCHS: 100,
+    OptimizerParams.LEARNING_RATE: 0.1,
+    OptimizerParams.WEIGHT_DECAY: 5e-4,
+    DataParams.SPLIT_RATIO: 0.8,
+    MemgraphParams.NODE_ID_PROPERTY: "id",
+    OtherParams.DEVICE_TYPE: "cpu",
+    TrainParams.CONSOLE_LOG_FREQ: 5,
+    TrainParams.CHECKPOINT_FREQ: 5,
+    TrainParams.BATCH_SIZE: 64,
+    ModelParams.AGGREGATOR: "mean",
+    DataParams.METRICS: [
+        "loss",
+        "accuracy",
+        "f1_score",
+        "precision",
+        "recall",
+        "num_wrong_examples",
+    ],
+    HeteroParams.OBSERVED_ATTRIBUTE: "",
+    HeteroParams.FEATURES_NAME: "features",
+    HeteroParams.CLASS_NAME: "class",
+    HeteroParams.REINDEXING: {},
+    HeteroParams.INV_REINDEXING: {},
+    OtherParams.PATH_TO_MODEL: "pytorch_models/model",
+    OtherParams.PATH_TO_MODEL_LAST: "pytorch_models/model_last",
+    OtherParams.PATH_TO_MODEL_SECOND_LAST: "pytorch_models/model_second_last",
+    OtherParams.PATH_TO_MODEL_THIRD_LAST: "pytorch_models/model_third_last",
+}
+
+
+##############################
+# set model parameters
+##############################
+
+
+def declare_data(ctx: mgp.ProcCtx):
+    """This function initializes global variable data.
+
+    Args:
+        ctx (mgp.ProcCtx): current context
+    """
+    global DEFAULT_VALUES
```

Comment:
Since DEFAULT_VALUES is not constant, it shouldn't be in upper case here

Reply:
Lowercase now.